src/functions/uploadToRSIS/application/data-mapper/__tests__/helpers/cat-a-mod1/inputs/fully-populated-inputs.ts

```diff
@@ -0,0 +1,149 @@
+      preTestDeclarations: {
+        insuranceDeclarationAccepted: true,
+        residencyDeclarationAccepted: true,
+        preTestSignature: '**DUMMY**',
+        DL196CBTCertNumber: '123456',
+      },
+      testData: {
+        singleFaultCompetencies: {},
+        drivingFaults: {
+          precautions: 5,
+          moveOffSafety: 3,
+          moveOffControl: 2,
+        },
+        dangerousFaults: {},
+        seriousFaults: {},
+        emergencyStop: {
+          speedNotMetSeriousFault: false,
+          firstAttempt: 50,
+        },
+        avoidance: {
+          speedNotMetSeriousFault: false,
+          firstAttempt: 50,
+        },
+        ETA: {},
+      },
+      testSummary: {
+        routeNumber: 88,
+      },
+    },
+  };
+}
+
+export function getCatAM1FullyPopulatedSeriousFaults(result: any): ResultUpload {
+  return {
+    ...result,
+    testResult: {
+      preTestDeclarations: {
+        insuranceDeclarationAccepted: true,
+        residencyDeclarationAccepted: true,
+        preTestSignature: '**DUMMY**',
+        DL196CBTCertNumber: '123456',
+      },
+      testData: {
+        singleFaultCompetencies: {},
+        drivingFaults: {},
+        dangerousFaults: {},
+        seriousFaults: {
+          precautions: true,
+          moveOffSafety: true,
+          moveOffControl: true,
+        },
+        emergencyStop: {
+          speedNotMetSeriousFault: false,
+          firstAttempt: 50,
+        },
+        avoidance: {
+          speedNotMetSeriousFault: false,
+          firstAttempt: 50,
+        },
+        ETA: {},
+      },
+      testSummary: {
+        routeNumber: 88,
+      },
+    },
+  };
+}
+
+export function getCatAM1FullyPopulatedDangerousFaults(result: any): ResultUpload {
+  return {
+    ...result,
+    testResult: {
+      preTestDeclarations: {
+        insuranceDeclarationAccepted: true,
+        residencyDeclarationAccepted: true,
+        preTestSignature: '**DUMMY**',
+        DL196CBTCertNumber: '123456',
+      },
+      testData: {
+        singleFaultCompetencies: {},
+        drivingFaults: {},
+        dangerousFaults: {
+          precautions: true,
+          moveOffSafety: true,
+          moveOffControl: true,
+        },
+        seriousFaults: {},
+        emergencyStop: {
+          speedNotMetSeriousFault: false,
+          firstAttempt: 50,
+        },
+        avoidance: {
+          speedNotMetSeriousFault: false,
+          firstAttempt: 50,
+        },
+        ETA: {},
+      },
+      testSummary: {
+        routeNumber: 88,
+      },
+    },
+  };
+}
+
+export function getCatAM1FullyPopulatedSingleFaultCompetencies(result: any): ResultUpload {
+  return {
+    ...result,
+    testResult: {
+      preTestDeclarations: {
+        insuranceDeclarationAccepted: true,
+        residencyDeclarationAccepted: true,
+        preTestSignature: '**DUMMY**',
+        DL196CBTCertNumber: '123456',
+      },
+      testData: {
+        singleFaultCompetencies: {
```

Comment:
After the changes that @edwardwilson introduced, the `singleFaultCompetencies` should contain `avoidance` & `emergencyStop` as well as they need to be present in `testData`